test/e2e/helpers.go

```diff
@@ -178,6 +180,7 @@ type WaitForServiceAvailableInput struct {
 func WaitForServiceAvailable(ctx context.Context, input WaitForServiceAvailableInput, intervals ...interface{}) {
 	namespace, name := input.Service.GetNamespace(), input.Service.GetName()
 	Byf("waiting for service %s/%s to be available", namespace, name)
+	Logf("waiting for service %s/%s to be available", namespace, name)
```

Comment:
Would it make sense to also add some logging around the time taken for the resources to be created? We can probably deduce it from other logs, just that it makes it easier for debugging.

Reply:
let's do this later, also want to add more logs in other places - I don't want to slow down this PR since it's mitigating the failing PR tests

Reply:
https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/1263